logfield.go

```diff
@@ -0,0 +1,136 @@
+package opentracing
```

Comment:
I was thinking we could put this into `opentracing/log` package, so that the fields could be created with `log.String()`, `log.Int64()`, etc. instead of much more verbose `opentracing.LogString()`